packages/genesys-spark-components/src/components/beta/gux-tooltip-pointer-beta/gux-tooltip-pointer.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Component,
+  Element,
+  h,
+  Host,
+  JSX,
+  Method,
+  Prop,
+  State,
+  Watch
+} from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxTooltipAccent } from '../gux-tooltip-beta/gux-tooltip-types';
+import { findElementById } from '@utils/dom/find-element-by-id';
+import { randomHTMLId } from '@utils/dom/random-html-id';
+import { Placement } from '@floating-ui/dom';
+
+/**
+ * @slot content - Slot for content
+ */
+@Component({
+  tag: 'gux-tooltip-pointer-beta',
+  shadow: true
+})
+export class GuxTooltipPointer {
+  @Element()
+  private root: HTMLElement;
+  private baseTooltip: HTMLGuxTooltipBaseBetaElement;
+  private id: string = randomHTMLId('gux-tooltip-pointer');
+  private tooltipObserver: MutationObserver;
+
+  @State()
+  private forElement: HTMLElement;
+
+  @State()
+  private role: string;
+
+  /**
+   * Indicates the id of the element the popover should anchor to. (If not supplied the parent element is used)
+   */
+  @Prop()
+  for: string;
+
+  @Prop()
+  accent: GuxTooltipAccent = 'light';
+
+  @Prop()
+  placement: Placement;
+
+  /**
+   * Determines whether the text in the tooltip is read by screenreaders.
+   * Use for cases where the forElement component handles the accessibility.
+   */
+  @Prop()
+  visualOnly: boolean = false;
+
+  /*
+   * Show tooltip
+   */
+  @Method()
+  async showTooltip(): Promise<void> {
+    return await this.baseTooltip.showTooltip();
+  }
+
+  /*
+   * Hide tooltip
+   */
+  @Method()
+  async hideTooltip(): Promise<void> {
+    return await this.baseTooltip.hideTooltip();
+  }
+
+  @Watch('for')
+  private updateForElement(): void {
+    this.forElement = this.getForElement();
+  }
+
+  componentWillLoad(): void {
+    trackComponent(this.root);
```

Comment:
Maybe track `placement` as variant here.

Reply:
ah yeah I didn't have placement at first so I removed the variant. I'll add it back in